tests/foreman/ui/test_errata.py

```diff
@@ -209,15 +209,15 @@ def repos_collection(module_org):
 
 
 @pytest.fixture(scope='function')
-def vm(repos_collection, rhel7_contenthost, default_sat):
+def vm(repos_collection, rhel7_contenthost, target_sat):
     """Virtual machine registered in satellite"""
-    repos_collection.setup_virtual_machine(rhel7_contenthost, default_sat)
-    rhel7_contenthost.add_rex_key(satellite=default_sat)
+    repos_collection.setup_virtual_machine(rhel7_contenthost, target_sat)
+    rhel7_contenthost.add_rex_key(satellite=target_sat)
     yield rhel7_contenthost
 
 
 @pytest.mark.tier3
-def test_end_to_end(session, module_org, module_repos_col, vm, default_sat):
+def test_end_to_end(session, module_org, module_repos_col, vm):
     """Create all entities required for errata, set up applicable host,
```

Comment:
@akhil-jha @JacobCallahan  Will this test work without calling target_sat?

Reply:
Yep! `module_org`, and other fixtures, depend on `target_sat`, so one will be available